index.html

```diff
@@ -87,7 +87,20 @@ <h3>TTML2 Subset</h3>
       document.</p>
 
       <p>[[!ttml-imsc1.0.1]] syntax that is not supported in [[!TTML2]] SHALL be mapped to an equivalent [[!TTML2]] syntax and
-      deprecated in [[!ttml-imsc1.1]].</p>
+      deprecated in [[!ttml-imsc1.1]], with the exception of the following extensions relative to the IMSC 1.0 Extension
```

Comment:
Since the features listed are not now supported in TTML2 I think the list is now the complete list not the exceptions?

Reply:
These are requirements, and I think the TTWG has heard from key users that these specific feature should remain in IMSC1.1 regardless of their disposition in TTML2.

Reply:
Yes sure, I'm not arguing with the list. My point is that the sentence above says "syntax that is not supported in TTML2 ... with the exception of the following extensions" but those are not exceptions, since they are in the set of syntax not supported in TTML2.

Reply:
I think the text is correct. Let's take #forcedDisplay as example. The exception is not that they the syntax/feature is not "supported in TTML2". The exception is that the the  "SHALL be mapped to an equivalent [TTML2] syntax and be deprecated" requirement do not apply.

Reply:
@tairt That's not how I read the text - I read it that the exception is relative to the set of IMSC 1.0.1 syntax that is not supported in TTML2. Since we have two different readings, that is evidence that the wording needs editorial work to resolve the ambiguity.